### ISSUE TITLE
Not rendering OTF correctly

### ISSUE THREAD

**Issue by kuckir** [opentypejs/opentype.js#117]
Hi, I have this font file http://www.ivank.net/AbrilFatface-Regular.otf and opentype.js does not render characters from Latin Extended-A group. However, charToGlyph() returns a glyph.

Here is an image: http://www.ivank.net/abril.png

**Comment by kuckir** [comment]
Hi, can somebody take a look at it? I really need some library to render OTF fonts.

**Comment by photopea** [comment]
Ok, so I am writing my own parser and I ran into the similar issue.

For a letter "ř", U+0159, index 291 in the previous font, there are following commands:
[44, 136, 154, 247, 6, 247, 99, 14]
44=95, 136=-3, 154=15, (247,6)=114, (247,99)=207, 14 = operator endchar

Do you know, how should be these commands interpreted, to draw "ř" ?

**Comment by Connum** [comment]
Here's the text in question for anyone trying to debug/fix this in the future: Příliš žluťoučký kůň úpěl ďábelské ódy

**Comment by Connum** [comment]
I just had a look at this font - that font does have the glyphs defined, but they contain no path data when opened with our library:
![image](https://github.com/opentypejs/opentype.js/assets/13076806/68de08bd-96fa-4194-a13d-f79aa4c0c5c0)
![image](https://github.com/opentypejs/opentype.js/assets/13076806/4699c12a-2883-4457-baac-78a2019567bc)
![image](https://github.com/opentypejs/opentype.js/assets/13076806/577062e3-ae0c-4972-bdef-a0de87fc01df)
Fontforge does render them
![image](https://github.com/opentypejs/opentype.js/assets/13076806/b3328b77-6d8f-4b79-9ab8-d7b31ea3b6cb)
but outputs a bunch of warnings when opening the font:
```
...
rcaron's SEAC-like endchar operator is deprecated for Type2
sacute's SEAC-like endchar operator is deprecated for Type2
scircumflex's SEAC-like endchar operator is deprecated for Type2
scaron's SEAC-like endchar operator is deprecated for Type2
...
```

So it looks like this could be the culprit. Even though it's deprecated, we should get these paths like other font software does.

**Comment by Connum** [comment]
The glyphs in question are also composite glyphs, so it seems like we don't support those under specific circumstances...

**Comment by Connum** [comment]
This test case is now part of https://github.com/unicode-org/text-rendering-tests. For opentype.js, it will be fixed via #633
